builtin_interfaces/Time stamp

# states
bool is_valid_max_distance_deviation
bool is_rolling_back
bool is_over_velocity
bool is_valid_latency

# values
float64 max_distance_deviation
float64 target_vel
float64 vehicle_vel
float64 latency

int64 invalid_count
